wisdom-war/src/components/inputs/CategoryDropdown.tsx

```diff
@@ -1,14 +1,19 @@
 import React, { useState } from 'react';
 import { RiArrowDropDownLine } from 'react-icons/ri';
 
-const CategoryDropdown: React.FC = () => {
+interface CategoryDropdownProps {
+  onChange: (value: string) => void; // Prop to handle category selection
+}
+
+const CategoryDropdown: React.FC<CategoryDropdownProps> = ({ onChange }) => {
   const [selectedCategory, setSelectedCategory] = useState<string>('Select Category');
   const [isOpen, setIsOpen] = useState<boolean>(false); // State to manage dropdown visibility
 
   const categories = ['FAC', 'Music', 'Movies', 'Art'];
```

Comment:
This needs to be connected to the backend to retrieve the Categories. We can reuse the code I wrote last week to get all the categories available in the quizzes. 
Also, we would need another option so the users can add more categories.